### ISSUE TITLE
Accessibility guidance suggests TabIndex should not be used

### ISSUE THREAD

**Issue by rachelkang** [MicrosoftDocs/windows-uwp#4369]
TabIndex often creates for more inaccessible experiences, and should only be used in rare exceptional scenarios. In those cases, it is advised that only 0 and negative values, but never positive values are used.

The docs should be updated to reflect this accessibility standard and encourage better accessibility practices https://github.com/MicrosoftDocs/windows-dev-docs/blob/docs/hub/apps/design/accessibility/keyboard-accessibility.md

**Comment by GrantMeStrength** [comment]
Closing this issue because it has been open for more than three years and does not appear to require current action. If this is still relevant, please open a new issue with updated details.
